pingouin/tests/test_pairwise.py

```diff
@@ -539,6 +547,7 @@ def test_pairwise_corr(self):
         pairwise_corr(data, covar=[('Psycho', 'Anxiety')])
         pairwise_corr(data, columns=[('Behavior', 'Rating')],
                       covar=[('Psycho', 'Anxiety')])
+        pairwise_corr(data, covar=data.columns)
```

Comment:
@JoElfner Can we add a test against the pingouin.pcorr() function? Just want to make sure that the r values we get are consistent with the latter.

Reply:
Sure! done